R/deployApp.R

```diff
@@ -642,13 +729,26 @@ applicationDeleted <- function(client, deployment, recordPath, appMetadata) {
   unlink(path)
 
   accountDetails <- accountInfo(deployment$account, deployment$server)
-  client$createApplication(
-    deployment$name,
-    deployment$title,
-    "shiny",
```

Comment:
This seems strange... should it be appMetadata$appMode?

Reply:
The "shiny" value is used as the application `template` for shinyapps.io; unused by Connect and Posit Cloud (on main).

What is the purpose of the `template`?